<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/git.png</normaloff>:/icons/git.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="CommitTable_text">
     <property name="text">
      <string>Commits:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="commit_current_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="prefix">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commit_search_text">
       <property name="placeholderText">
        <string>Filter: Enter a keyword about the commit description.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="CommitTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Author</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Previous</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Branch</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Branch list:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="BranchList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="branch_current">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="branch_checkout">
         <property name="statusTip">
          <string>Switch and load the latest commit of this branch.</string>
         </property>
         <property name="text">
          <string>Checkout</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/icons/git.png</normaloff>:/icons/git.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="branch_delete">
         <property name="statusTip">
          <string>Remove all the commits on this branch.</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="AuthorList_text">
     <property name="text">
      <string>Author list:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="AuthorList"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="FileAuthor_text">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FileAuthor"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="FileDescription_text">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="FileDescription">
     <property name="placeholderText">
      <string>Explain what you have done here (required).</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="commit_add">
       <property name="statusTip">
        <string>Save the changes for this phase and provide backtracking points for this phase.</string>
       </property>
       <property name="text">
        <string>Commit</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/git.png</normaloff>:/icons/git.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="branch_add">
       <property name="statusTip">
        <string>Creat a new branch then save the commit to this branch.</string>
       </property>
       <property name="text">
        <string>New Branch</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/git.png</normaloff>:/icons/git.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="commit_stash">
       <property name="statusTip">
        <string>Abandon all changes of this time, and return to the latest commit.</string>
       </property>
       <property name="text">
        <string>Stash</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
